### PR TITLE
Individual project proposal

### PR DESCRIPTION
**Pull request by Consti10**

### CODE REVIEW

File: projects/constantin/proposal.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+written a) in .ts and b) using a h.e.d.s as input instead of a nromal triangle mesh.
+
+
+## Implementation
+
+I plan to use the exercise template as a starting point and therefore implement this task in .ts. My milestones are:
+1) naive implementation of the voxelizing algorithm in .ts.
+2) improving the algorithm using properties of h.e.d.s.
+3) visualizing the results nicely
+4) adding additional features like coloring the resulting voxel mesh, variable cube size and calculating normals.
```
Comment (JohannesMerkt) [inline projects/constantin/proposal.md:33]: Are you planning to have the cube voxels be connected to one single mesh surface?

Comment (MarlQ) [reply projects/constantin/proposal.md:33]: I'm also wondering, are the non-visible faces going to be deleted? 
From the wording, I'm assuming they won't.
Now I'm considering whether large voxelized meshes will cause lag in the viewport, and how much geometry is actually created in the end depending on the resolution.

Comment (nico778) [reply projects/constantin/proposal.md:33]: Are you using threejs or something else?

Comment (nico778) [reply projects/constantin/proposal.md:33]: Are you planning to use threejs or something else?

**Review by changkun — APPROVED** [review]
This is a classic algorithm of convering a given mesh to voxels and the voxels are also represented by mesh representation.

Most of the complexity comes from a careful handling of halfedge constructions, and matches the intention of this course.

Note that it might be easier to assume input format subjects to some constraints so that the implementation could be easier.

I found this project interesting and doable and hence looking forward to see its result.

**Pull request merged**